pygmt/tests/test_text.py

```diff
@@ -92,10 +93,48 @@ def test_text_nonexistent_filename():
     Run text by passing in a list of filenames with one that does not exist
     """
     fig = Figure()
-    with pytest.raises(GMTInvalidInput):
+    with pytest.raises(GMTCLibError):
         fig.text(region=[10, 70, -5, 10], textfiles=[POINTS_DATA, "notexist.txt"])
 
 
+@pytest.mark.mpl_image_compare
+def test_text_position(region):
+    """
+    Print text at center middle and four corners (top left/right, bottom
+    left/right) of map.
+    """
+    fig = Figure()
+    fig.text(region=region, projection="x1c", frame="a", position="CM", text="C M")
+    for position in ("TL", "TR", "BL", "BR"):
+        fig.text(frame="a", position=position, text=position)
```

Comment:
`frame="a"` is not needed.
```suggestion
        fig.text(position=position, text=position)
```